### ISSUE TITLE
Release: release notes and changelog

### ISSUE THREAD

**Issue by dexX7** [mastercoin-MSC/mastercore#290]
Ideally there would be some release notes and a change log similar to:

https://github.com/bitcoin/bitcoin/blob/0.9/doc/release-notes.md

---

@zathras-crypto said:

I started to try and work this, but it's an absolute PITA because we don't have a previous state to work from. I don't have a clear path to do this, I tried walking commits and doing a changelog like that but it seems entirely fruitless (eg a changelog that has "X added" then a few lines down "X removed" makes no sense because in the end X is unchanged).

Trying to think about how to start a changelog half way through a project where one hasn't existed before - open to suggestions on process for generating this documentation - anyone with experience of doing this type of thing want to weigh in here?

Basically the result I think we're trying to aim for is what has changed between this version and 0.0.9-rel and currently the only way I can think to do this is to step through all the work we've done and try and work it all out manually via diffing the 0.0.9.1 code against 0.0.9 code instead of using commit history, but that seems like an awfully long winded process :(

**Comment by dexX7** [comment]
@zathras-crypto: agreed, copying the commit messages one to one doesn't seem very fitting in this case, but anyway, this should help:

https://github.com/mastercoin-MSC/mastercore/compare/mscore-0.0.9...zathras-crypto:0.0.9.1-Z

**Edit:** using proper tags in your repo was extremely nice to have, so as further resource one may also take a look at the issue section:

https://github.com/zathras-crypto/mastercore/issues?q=label%3A%22ready+to+close%22

Ideally, any PR, issue, etc. should be tagged in a similar fashion in the future.

**Comment by zathras-crypto** [comment]
Took a stab - https://github.com/zathras-crypto/mastercore/blob/a172dc402856ae192c6ec09f8a75e2dd97825d21/doc/release-notes.md but still doesn't feel right - I mean technically https://github.com/mastercoin-MSC/mastercore/pull/243 went in after 0.0.9-rel tag so that should all be there in the changelog too perhaps?  Or since this is our first release of the UI do we need to have a detailed changelog of changes to UI in this release or just going forward?

**Comment by dexX7** [comment]
If #243 was not part of the last release, then it should be included.

I think you did a very good job! My (very few) notes are inline. It might make sense to create a new file though, instead of overwriting the Bitcoin Core release notes, which _also_ apply here.

**Comment by dexX7** [comment]
The change logs for 0.0.9.1 are finished and any follow up should be created for https://github.com/OmniLayer/omnicore.
